holmes/plugins/toolsets/__init__.py

```diff
@@ -8,31 +8,32 @@
 
 import holmes.utils.env as env_utils
 from holmes.common.env_vars import (
-    USE_LEGACY_KUBERNETES_LOGS,
     DISABLE_PROMETHEUS_TOOLSET,
+    USE_LEGACY_KUBERNETES_LOGS,
 )
 from holmes.core.supabase_dal import SupabaseDal
 from holmes.core.tools import Toolset, ToolsetType, ToolsetYamlFromConfig, YAMLToolset
 from holmes.plugins.toolsets.atlas_mongodb.mongodb_atlas import MongoDBAtlasToolset
 from holmes.plugins.toolsets.azure_sql.azure_sql_toolset import AzureSQLToolset
 from holmes.plugins.toolsets.bash.bash_toolset import BashExecutorToolset
 from holmes.plugins.toolsets.coralogix.toolset_coralogix import CoralogixToolset
+from holmes.plugins.toolsets.datadog.toolset_datadog_general import (
```

Comment:
These package import sequence change can be ignored if we introduce https://github.com/HolmesGPT/holmesgpt/pull/1252